code/modules/surgery/organs/dwarf_organ.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/organ/alcoholvessel //essentially the opposite of the xeno's plasmavessel, but with alcohol
```

Comment:
yeah but I also don't want it to be locked into a single species and allow people to steal it for their own dwarf-based alcoholism and mining speeds.